sdk/storage/Azure.Storage.Queues/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.5.0-preview.1 (Unreleased)
-
+## 12.5.0-preview.1 (2020-09-30)
+- This preview contains bug fixes to improve quality.
```

Comment:
should this mention service version ?

Reply:
I think we still technically call a service version from 2018 for Queues, there has never been an update.  Even when users specify a newer service version with `QueueClientOptions.ServiceVersion`, they all actually point at the same version.